Mathlib/Dynamics/Circle/RotationNumber/TranslationNumber.lean

```diff
@@ -441,6 +443,7 @@ theorem iterate_mono {f g : CircleDeg1Lift} (h : f ≤ g) (n : ℕ) : f^[n] ≤
   iterate_monotone n h
 #align circle_deg1_lift.iterate_mono CircleDeg1Lift.iterate_mono
 
+@[gcongr]
```

Comment:
Do you mind splitting this and all other changes that don't require `PointwiseLE` to be defined to another PR? The diff is currently quite noisy.

Reply:
Done.